source/states/PlayCollisionState.hx

```diff
@@ -66,4 +84,13 @@ class PlayCollisionState extends FlxState
 			bgColor = FlxColor.fromRGB(20, 20, 20);
 		}
 	}
+
+	private function checkHit(a:FlxSprite, b:FlxSprite):Bool {
+		return FlxG.pixelPerfectOverlap(a, b);
```

Comment:
Not that I care too much, but I imagine pixelPerfect is on the expensive side.  Do we need it?  Or can we get away with simple radius collisions?

Reply:
This only gets called if the broad-stroke collision (aka AABB collisions) detect something. So the process goes like:

1. `FlxG.overlaps` checks FlxSprite hitbox collisions between the first two args (in our case, FlxGroups)
2. Any Collisions it detects, it hands off to the `checkHit` function to give it a pass/fail
3. Any that pass `checkHit` (i.e. return `True` from that function) then are sent to the `damage` function to actually do our collision logic.

So yes, while it is expensive, it is only being performed when a potential collision is already identified.... so maybe it's ok for now? (Let's keep an eye on it)

Reply:
`.......collide this with  this, and do this if this says it was for sure a collision`
`................|..........|............|.......|`
`................V..........V............V.......V`
`FlxG.overlap(enemies, projectiles, damage, checkHit);`